Sources/SuperDelegate+Handoff.swift

```diff
@@ -47,31 +46,31 @@ extension SuperDelegate {
     // MARK: UIApplicationDelegate
     
     
-    @warn_unused_result
-    final public func application(application: UIApplication, willContinueUserActivityWithType userActivityType: String) -> Bool {
+    final public func application(_ application: UIApplication, willContinueUserActivityWithType userActivityType: String) -> Bool {
         guard let userActivityCapableSelf = self as? HandoffCapable else {
-            noteImproperAPIUsage("Received willContinueUserActivityWithType but \(self) does not conform to HandoffCapable. Not handling handoff event.")
+            noteImproperAPIUsage(text: "Received willContinueUserActivityWithType but \(self) does not conform to HandoffCapable. Not handling handoff event.")
```

Comment:
Since noteImproperAPIUsage takes just one argument, I think the guidelines would have you omit the argument here.